code/game/machinery/computer/apc_control.dm

```diff
@@ -153,12 +183,12 @@
 			usr.log_message("set the logs of [src] [should_log ? "On" : "Off"].", LOG_GAME)
 		if("restore-console")
 			restoring = TRUE
-			addtimer(CALLBACK(src, PROC_REF(restore_comp)), rand(3,5) * 9 SECONDS)
+			addtimer(CALLBACK(src, PROC_REF(restore_comp), usr), rand(3,5) * 9 SECONDS)
```

Comment:
```suggestion
			addtimer(CALLBACK(src, PROC_REF(restore_comp), ui.user), rand(3,5) * 9 SECONDS)
```
not to be a pain but id appreciate these changes too. (alternatively, since we're using `ui.user` alot, u could cache it into a /mob/living reference before entering the switch statement and use that instead, but it doesnt matter much)

Reply:
it's best to be a pain than leave these bits to the next guy that may not come